docs/running/introduction.rst

```diff
@@ -71,7 +70,7 @@ The :ref:`clusterRef` are command line tools used to provision nodes in your des
 They allows you to launch nodes, ssh to the leader, and rsync files back and forth.
 
 For detailed instructions for using the provisioner see :ref:`runningAWS` or :ref:`runningGCE`. An
-Azure provisioner is in the works and coming soon. For more details see the `Azure provisioner github ticket`_.
+Azure provisioner is working but has limited support. For more details see the `Azure provisioner github ticket`_.
```

Comment:
I would just say limited support and leave out the `working` part since we can't really know without CI/CD in place for it.